src/test/resources/data/githubRefresh.csv

```diff
@@ -0,0 +1,3 @@
+repository,gitReference,username,expectedStatus
+coverbeck/viral-pipelines,refs/heads/dockstore-test2,coverbeck,204
```

Comment:
should probably be moved/forked

Reply:
It's a fork of the Broad's, where I updated the .dockstore.yml to not have dozens of workflows, because it was slowing down the tests too much. But good point, I'll pick a different workflow not in my own repo.

Reply:
c420e8bbf66955a9c415ca18740b12dd557b01e8